README.md

```diff
@@ -11,157 +18,79 @@ Client:
-
-```javascript
-{
-  "watch": ["src/server/"]
-}
+```
+ "check": - runs lint, format and stylelint scripts together,
+ "clear-demo": - clears demo data from the database,
+ "build:dev": - builds for development,
+ "build:prod": - builds for production,
+ "dev": - runs development server,
+ "format": - formats all the files using Prettier,
+ "format:fix": - formats and fix all the files using Prettier,
+ "lint": - lint all the files using eslint,
+ "lint:fix": - lint and fix all files using eslint,
+ "migrate": - runs migration script,
+ "migrate:up":  - migrate database up,
+ "migrate:down":  - migrate database down,
+ "seed": - seed database with demo data,
+ "start:client:dev": - start client dev server,
+ "start:server:dev": - start dev server,
+ "start:server:prod": - start production server,
+ "stylelint": - stylelint all style files,
+ "stylelint:fix": - stylelint and fix all style files,
+ "test": - run tests
 ```
 
-Here, we tell nodemon to watch the files in the directory src/server where out server side code resides. Nodemon will restart the node server whenever a file under src/server directory is modified.
-
-### Express
-
-Express is a web application framework for Node.js. It is used to build our backend API's.
-src/server/index.js is the entry point to the server application.
-
-### Concurrently
-
-[Concurrently](https://github.com/kimmobrunfeldt/concurrently) is used to run multiple commands concurrently. I am using it to run the webpack dev server and the backend node server concurrently in the development environment. Below are the npm/yarn script commands used.
+## Database migrations
 
-```javascript
-"client": "webpack-dev-server --mode development --devtool inline-source-map --hot",
-"server": "nodemon src/server/index.js",
-"dev": "concurrently \"npm run server\" \"npm run client\"",
-```
+To perform DB migrations we use `migrate-mongo` tool. To config migrate-mongo edit migrate-mongo-config.js file. To create migration file run `npm run migrate` create <version> in your terminal. Migration file will be created at server/migrations directory. To run all new migrations run `npm run migrate:up` or `npm run migrate up` in your terminal. To rollback, last migration run `npm run migrate:down` or `npm run migrate down` in your terminal. To see migrations status run `npm run migrate status` in your terminal.
 
-### ESLint + Prettier + Husky
+## Contributing
 
-[ESLint](https://eslint.org/) takes care of the code-quality. [Prettier](https://prettier.io/) takes care of all the formatting. [Husky](https://github.com/typicode/husky) takes care of running linting rules before we commit or push the code.
+Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
 
-### License and trademarks info
+Please make sure to update tests as appropriate.
 
-This project uses:
+## License and trademarks info
```

Comment:
What about EOC license?

Reply:
We need to ask Lukasz for this

Reply:
OK, we can add license info in badges:
![Screenshot 2019-09-02_12-39-51-329](https://user-images.githubusercontent.com/25374390/64109272-6032e680-cd7f-11e9-9fb3-986273f5a645.png)